diff --git a/src/language_server/state.rs b/src/language_server/state.rs
index bc6577a..0dce064 100644
--- a/src/language_server/state.rs
+++ b/src/language_server/state.rs
@@ -269,6 +269,8 @@ fn build_database_configuration(root: &Path, configuration: &Configuration) -> D
 
     let includes: Vec<Cow<'static, [u8]>> =
         source.includes.iter().cloned().map(|s| Cow::<'static, [u8]>::Owned(s.into_bytes())).collect();
+    let patches: Vec<Cow<'static, [u8]>> =
+        source.patches.iter().cloned().map(|s| Cow::<'static, [u8]>::Owned(s.into_bytes())).collect();
     let extensions: Vec<Cow<'static, [u8]>> = if source.extensions.is_empty() {
         vec![Cow::Borrowed(b"php")]
     } else {
@@ -299,7 +301,7 @@ fn build_database_configuration(root: &Path, configuration: &Configuration) -> D
         includes,
         excludes,
         extensions,
-        patches: Vec::new(),
+        patches,
         glob: source.glob.to_database_settings(),
     }
 }
